webapp/src/routes/home/game/index.js

```diff
@@ -17,12 +21,35 @@ export class Game {
     this.htmlElement.append(this._app.renderer.view);
 
     this.background = new Background({ width: this.width, height: this.height });
-    this.warehouse = new Warehouse({ rendererWidth: this.width, rendererHeight: this.height, scale: 1/3 });
+    this.warehouse = new Warehouse({ rendererWidth: this.width, rendererHeight: this.height, scale: 1 / 3 });
```

Comment:
z czego wynika akurat taki a nie inny scale? wielkość obrazka?

Reply:
Na retinie potrzeba większej rozdzielczości, w związku z czym przygotowałem assety x3, ale wyświetlają się tez x3, więc musiałem je o tyle zmniejszyć

Reply:
przgadałem to jeszcze z @mkmieciak żeby się upewnić: robienie scale'a w ten sposób nie do końca robi to co byśmy chcieli na retinie. Lepiej wewnątrz klasy warehouse ustawić obrazkowi odpowiedni statyczny width i height. Dzięki temu, ze w ramach tworzenia klasy Application mamy podane `ration: window.devicePixelRation` i `autoResize: true` to się automatycznie odpowiednio zeskaluje na retinie